### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by harshkotkar5557**
Preview
grey-tex

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -56,6 +56,12 @@
     --font-bold:700
 }
 
+.d-none{
+    display: none !important;
+}
+.d-block{
+    display: block !important;
```
Comment (vaibhav18matere) [inline styles.css:63]: try to avoid `!important` as much as possible. Find alternatives to doing it.

**Review by vaibhav18matere — APPROVED** [review]

File: alert/alert.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
         </header>
     </nav>
     <section class="middleSection">
-        <div class="siderbar">
+        <div class="siderbar" id="siderbar">
```
Comment (Mahi24-prog) [inline alert/alert.html:30]: you can give different naming convention for id and class.

File: text-utilities/text.html [hunk 3]
```diff
@@ -77,6 +77,18 @@ <h2 class="p-1">Text Alignment</h2>
...
+           <h2 class="p-1">Simple text utilities</h2>
+           <div class="card p-1 d-flex justify-around">
+                <div class="text-grey">Grey Text</div>
+               <div class="text-small">small Text</div>
+           </div>
+           <div class="doc-img">
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522text-grey%2522%253EGrey%2520Text%253C%252Fdiv%253E%250A%2520%253Cdiv%2520class%253D%2522text-small%2522%253Esmall%2520Text%253C%252Fdiv%253E%250A"
+            style="width: 100%; height: 242px; border:0; transform: scale(1); overflow:hidden;margin: 1rem auto;"
+            sandbox="allow-scripts allow-same-origin">
```
Comment (Mahi24-prog) [inline text-utilities/text.html:89]: For iframe give your own css style. Remove this default style. Default style is not responsive.

**Review by Mahi24-prog — APPROVED** [review]
Check suggestion I have given. You can add some more text utilities.

File: text-utilities/text.html (hunk 3 above)
Comment (harshkotkar5557) [reply text-utilities/text.html:89]: Okay

File: alert/alert.html (hunk 2 above)
Comment (harshkotkar5557) [reply alert/alert.html:30]: Okay

**Pull request merged**